CleverTapSDK/InApps/CTInAppDisplayManager.m

```diff
@@ -514,6 +514,12 @@ - (void)displayNotification:(CTInAppNotification*)notification {
 #endif
 }
 
+- (void)notifyNotificationDidShow:(CTInAppNotification *)notification {
+    if (self.inAppNotificationDelegate && [self.inAppNotificationDelegate respondsToSelector:@selector(inAppNotificationDidShow:)]) {
+        [self.inAppNotificationDelegate inAppNotificationDidShow:notification.jsonDescription];
```

Comment:
In Android they are passing CTInAppNotification object, not the jsonDescription: https://github.com/CleverTap/clevertap-android-sdk/blob/master/clevertap-core/src/main/java/com/clevertap/android/sdk/InAppNotificationListener.java#L31

We should pass the Dictionary only as implemented here, can you check on Android side if they are exposing CTInAppNotification

Reply:
The `CTInAppNotification` is a public class in Android SDK. The in-app notification delegate methods expose the `CTInAppNotification` object. 
We cannot expose the whole `CTInappNotification` class since there are private members. We will need to refactor it and have public and internal/private headers. We can then pass the `CTInappNotification` as parameter for `inAppNotificationDidShow:` and `shouldShowInAppNotification:`.
The json description is the next best thing we can offer since you can get the data you need.